goccm.go

```diff
@@ -120,5 +129,12 @@ func (c *concurrencyManager) WaitAllDone() {
 
 // RunningCount Returns the number of goroutines which are running
 func (c *concurrencyManager) RunningCount() int32 {
-	return c.runningCount
+	return atomic.LoadInt32(&c.runningCount)
+}
+
+// Release Closes all the channels associated with goccm
+func (c *concurrencyManager) Release() {
+	close(c.managerCh)
```

Comment:
Why not close channels in Close() method?

Reply:
First it was there and I also think that it is more natural. But I think you expect to call the close twice from this test:
https://github.com/zenthangplus/goccm/blob/master/goccm_test.go#L22